src/engine/renderer/tr_local.h

```diff
@@ -1324,9 +1324,18 @@ static inline void glFboSetExt()
 	  SHADER_LIGHT // light material: attenuation
 	};
 
+	// Size of MAX_QPATH + "$depth".
+	const int depthShaderNameLen = MAX_QPATH + 6;
```

Comment:
constexpr is the new cool. Also do you need +7 for null char?

since its a constant should we call it DEPTH_SHADER_NAME_LEN?

Reply:
The null terminator was already counted in MAX_QPATH, as a single string requires only one null terminator. Anyway I removed this code.